src/pcv/IPCVDepositV2.sol

```diff
@@ -11,4 +11,6 @@ interface IPCVDepositV2 is IPCVDeposit {
     function harvest() external;
 
     function accrue() external returns (uint256);
+
+    function lastRecordedProfit() external returns (uint256);
```

Comment:
We keep on coming back to this point, however if we go with this architecture, all places where an individual PCV deposit balance is queried, we instead should be querying the PCVOracle, which isn't correct in some cases where you need a value that returns the raw decimals amount instead of the normalized decimals amount.

Additionally, this means that the PCVOracle needs a separate mode to read from deposits that governance has marked down. If we want to do that, we should do that refactor in a separate PR from this feature and agree on scope and architecture beforehand.